cloudinary-http42/src/test/java/com/cloudinary/test/StructuredMetadataTest.java

```diff
@@ -0,0 +1,4 @@
+package com.cloudinary.test;
+
+public class StructuredMetadataTest extends AbstractStructuredMetadataTest {
```

Comment:
Why "structured" all of a sudden?

Reply:
The word Metadata is already used in the api (search API). In places where we have enough context (MetadatField, MetadataRule) it seems to me 'structured' can be omitted. For this test name it's a bit ambiguous. Any thoughts @jackieros?